src/components/PersonLink.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  persons: Person[];
+  personSlug : string
+};
+
+export const PersonLink:FC<Props> = ({ person, persons, personSlug }) => {
```

Comment:
Person link, as is infered from the name of the component, should be only the link to a person by name. It shall not encompass whole row markup and excessive, not realted logic. Try to adhere to "Single Responsibility" priciple and avoid "God Object" anti-pattern

Reply:
Thank you. Really good advice!